Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -3201,8 +3201,12 @@ - (void)_updateFixedColorExtensionViews
             _fixedColorExtensionViews.setAt(side, extensionView);
         }
 
-        RetainPtr predominantColor = cocoaColorOrNil(_fixedContainerEdges.predominantColor(side));
-        [extensionView updateColor:predominantColor.get() ?: self.underPageBackgroundColor];
+        if (_showsColorInToolbar || side != WebCore::BoxSide::Top) {
+            RetainPtr predominantColor = cocoaColorOrNil(_fixedContainerEdges.predominantColor(side));
+            [extensionView updateColor:predominantColor.get() ?: self.underPageBackgroundColor];
+        } else
+            [extensionView updateColor:nil];
```

Comment:
`-updateColor` wasn't really intended to take `nil`:

```
- (void)updateColor:(WebCore::CocoaColor *)color
{
    [self _updateColor:color visible:YES];
}
```

Would it work to use `-fadeOut` instead?